conans/test/unittests/client/generators/cmake_find_package_multi_test.py

```diff
@@ -27,3 +28,27 @@ def test_cmake_find_package_multi_version(self):
         content = generator.content
         config_version = content["my_pkg-config-version.cmake"]
         self.assertIn('set(PACKAGE_VERSION "0.1")', config_version)
+
+
+def test_cmake_find_package_multi_links_flags():
+    # https://github.com/conan-io/conan/issues/8703
+    conanfile = ConanFile(Mock(), None)
+    conanfile.settings = "os", "compiler", "build_type", "arch"
+    conanfile.initialize(Settings({"os": ["Windows"],
+                                   "compiler": ["gcc"],
+                                   "build_type": ["Release"],
+                                   "arch": ["x86"]}), EnvValues())
+    conanfile.settings.build_type = "Release"
+    conanfile.settings.arch = "x86"
+
+    cpp_info = CppInfo("mypkg", "dummy_root_folder1")
+    cpp_info.sharedlinkflags = ["/NODEFAULTLIB", "/OTHERFLAG"]
+    cpp_info.exelinkflags = ["/OPT:NOICF"]
+    conanfile.deps_cpp_info.add("mypkg", cpp_info)
+
+    gen = CMakeFindPackageMultiGenerator(conanfile)
+    files = gen.content
+    d = files["mypkgTarget-release.cmake"]
+    assert "$<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,EXECUTABLE>:-/OPT:NOICF>" in d
+    assert "$<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,MODULE_LIBRARY>:-/NODEFAULTLIB;-/OTHERFLAG>" in d
```

Comment:
so, is it really expected to pass `-/OPT` with both dash and slash?

Reply:
Apparently it worked, but you are right, better double check, I'll check it.

Reply:
yeah, that syntax looks a bit weird to me

Reply:
Ok, it was working, it didn't raise an error, but just a warning. Dropped the extra /, please check.